### ISSUE TITLE
New splitChannelsOptions.overlay Argument Doesn't Work

### ISSUE THREAD

**Issue by DrLongGhost** [katspaugh/wavesurfer.js#2161]
### Wavesurfer.js version(s):
4.4.0

### Browser and operating system version(s):
Chrome/FF on Mac

### Code needed to reproduce the issue:
Params are as follows:
```
      const wavesurferArgs = {
        container: document.getElementById('wavesurferContainerInternal'),
        waveColor: '#28275f',
        progressColor: 'rgba(210, 239, 241)',
        cursorColor: '#66c8cf',
        normalize: true,
        pixelRatio: 2,
        splitChannels: true,
        splitChannelsOptions: {
          relativeNormalization: true, (also tried false)
          overlay: true,
          channelColors: {
            0: {
              progressColor: 'green',
              waveColor: 'pink'
            },
            1: {
              progressColor: 'orange',
              waveColor: 'purple'
            }
          }
        },
        src: this.wavesurferUrl,
        plugins,
      };
```

### Use behaviour needed to reproduce the issue:

I just noticed this feature was added 4 days ago. I was excited to try it out but I don't think it actually works.

Here are a couple screenshots illustrating what I'm seeing. Either this is user error of some sort, maybe I'm misunderstanding what "splitChannelsOptions.overlay" is supposed to do, or it is broken.

splitChannelsOptions.overlay = false

<img width="860" alt="VoiceVibes_Evaluate_Recording" src="https://user-images.githubusercontent.com/1702840/105051417-ba981a00-5a3c-11eb-9278-18705e8f1eda.png">

This shows 2 people talking. You can see the person captured in the right channel is speaking at the 50-60 second mark.

splitChannelsOptions.overlay = true 

<img width="856" alt="VoiceVibes_Evaluate_Recording" src="https://user-images.githubusercontent.com/1702840/105051623-f16e3000-5a3c-11eb-8df8-f63989cee7a7.png">

In the supposedly combined waveform, the second speaker/right channel is missing. I would expect to see the purple waveform at the 50-60 second mark.

**Comment by sundayz** [comment]
The only place where I can find `overlay` being refernced in the code. must be a bug.

https://github.com/katspaugh/wavesurfer.js/blob/658bd921be60a64192331ce8511bf1bce68cc4ec/src/drawer.multicanvas.js#L474

**Comment by johnpaulmedina** [comment]
I too am experiencing this bug. This is the first time I am working with the WaveSurfer library - did older versions have the overlay option? I don't mind making a PR if I can fix the issue.

**Comment by johnpaulmedina** [comment]
Digging a little deeper I was able to find where the issue is coming from but not yet have implemented a fix.

See screenshot
<img width="1024" alt="Screen Shot 2021-01-24 at 4 06 58 PM" src="https://user-images.githubusercontent.com/1139439/105643552-8074b100-5e5e-11eb-9c0e-4b08b88260bd.png">

`wavesurfer.js` line: `1227`

I outputted the _ref.offsetY value to console when overlay false and overlay true and noticed that the offset value was the same. I changed the `offsetY` value to 0 to see if it would change it and sure enough it did. I am going to do a little reverse engineering here to trace back when the value is set and sent to the method.
